features/user-agent-sniffing.yml

```
@@ -0,0 +1,6 @@
+name: User agent sniffing
+description: "The `navigator.userAgent` read-only property returns the user agent string for the current browser. Browser identification based on detecting the user agent string is unreliable and is not recommended."
```

Optional. I think we can be a bit more specific about what we don't want people to do with this and what they ought to do instead. Maybe like this?

```suggestion
description: "The `navigator.userAgent` read-only property returns the user agent string for the current browser. Selectively showing content based on the user agent string is unreliable. Consider using feature detection instead."
```